README.md

```diff
@@ -96,6 +96,7 @@ Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
 Please use the following to reach members of the community:
 
-- Slack: Join our [slack channel](https://cephcsi.slack.com)
+- Slack: Join our [slack channel](https://cephcsi.slack.com),
+  [invite link](https://bit.ly/32vcEfO )
```

Comment:
(nit) When viewed this comes in the same line, I would like more emphasis on this so that first time users (who are the only ones needing the link IMO) do not miss this. Otherwise thanks for the change.